src/modules/ens/sagas.ts

```diff
@@ -76,11 +76,9 @@ function* handleSetAlias(action: SetAliasRequestAction) {
   try {
     const client = new CatalystClient(PEER_URL, 'builder')
     const entities: Entity[] = yield client.fetchEntitiesByPointers(EntityType.PROFILE, [address.toLowerCase()])
-    const entity = entities.length > 0 ? entities[0] : null
+    const defaultEntity: Entity = yield getDefaultProfileEntity()
```

Comment:
Even this works, I think that we are using for consistency `yield call(....)`. Am I right? same for the line above.

Reply:
fixing!